src/scenes/AcceleratorRouter/ParticipantProjects/ListView.tsx

```diff
@@ -0,0 +1,154 @@
+          name: strings.LAND_USE_MODEL_TYPE,
+          type: 'string',
+        },
+      ]
+    : [];
+
+const fuzzySearchColumns = ['name'];
+const defaultSearchOrder: SearchSortOrder = {
+  field: 'name',
+  direction: 'Ascending',
+};
+
+export default function ListView(): JSX.Element {
+  const { activeLocale } = useLocalization();
+  const dispatch = useAppDispatch();
+  const snackbar = useSnackbar();
+
+  const [projects, setProjects] = useState<ParticipantProjectSearchResult[]>([]);
+  const [requestId, setRequestId] = useState<string>('');
+  const result = useAppSelector(selectParticipantProjectsListRequest(requestId));
+
+  useEffect(() => {
+    if (result?.status === 'error') {
+      snackbar.toastError();
+    }
+    if (result?.data) {
+      setProjects(result.data);
+    }
+  }, [result, snackbar]);
+
+  const dispatchSearchRequest = useCallback(
+    (locale: string | null, search: SearchNodePayload, sortOrder: SearchSortOrder) => {
+      if (locale) {
+        const request = dispatch(requestListParticipantProjects({ search, sortOrder }));
+        setRequestId(request.requestId);
+      }
+    },
+    [dispatch]
+  );
+
+  const cohorts = useMemo<Record<string, string>>(
+    () =>
+      (projects || []).reduce(
+        (acc, project) => {
+          acc[project.participant_cohort_id] = project.cohortName;
+          return acc;
+        },
+        {} as Record<string, string>
+      ),
+    [projects]
+  );
+
+  const featuredFilters: FilterConfig[] = useMemo(
+    () =>
+      activeLocale
+        ? [
+            {
+              field: 'participant_cohort_id',
+              id: 'participant_cohort_id',
+              label: strings.COHORT,
+              options: (projects || [])?.map(
+                (project: ParticipantProjectSearchResult) => `${project.participant_cohort_id}`
+              ),
+              pillValueRenderer: (values: (string | number | null)[]) =>
+                values.map((value) => cohorts[value || ''] || '').join(', '),
+              renderOption: (id: string | number) => cohorts[id] || '',
+            },
+            {
+              field: 'participant_cohort_phase',
+              id: 'participant_cohort_phase',
+              label: strings.PHASE,
+              options: (projects || [])?.map(
+                (project: ParticipantProjectSearchResult) => `${project.participant_cohort_phase}`
+              ),
+            },
+          ]
+        : [],
+    [activeLocale, cohorts, projects]
+  );
+
+  return (
+    <>
+      <TableWithSearchFilters
+        busy={result?.status === 'pending'}
+        columns={columns}
+        defaultSearchOrder={defaultSearchOrder}
+        dispatchSearchRequest={dispatchSearchRequest}
+        featuredFilters={featuredFilters}
+        fuzzySearchColumns={fuzzySearchColumns}
+        id='accelerator-participan-projects-table'
+        Renderer={CellRenderer}
+        rows={projects}
+        title={strings.PROJECTS}
+      />
+    </>
+  );
```

Comment:
Is the fragment wrapper necessary here?

Reply:
Good catch, not needed, there were other elements in there but I moved them into the Table component. Will remove it.